### PR TITLE
[cimb] Re-enable ruleset, removed non-functional domains.

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/270278377
`ERROR src/chrome/content/rules/Heroku.xml: Not enough tests (0 vs 2) for <Rule from '^http://([^/:@\.]+\.)?heroku(app)?\.com/' to 'https://\g<1>heroku\g<2>.com/'>`

**Comment by Bisaloo** [comment]
I fixed tests in https://github.com/EFForg/https-everywhere/pull/12275.

File: src/chrome/content/rules/CIMB-Malaysia.xml [hunk 1]
```diff
@@ -22,21 +13,20 @@ Fetch error: http://www.cimb-principal.com.my/ => https://www.cimb-principal.com
 		- (www.)cimbsecurities.com			(², CN: *.itradecimb.com.my, itradecimb.com.my)
 		- (www.)cwealthadvisors.com.my		(², CN: www.cimb-principal.com.my)
 		- (www.)cimbaviva.com				(site unreachable)
+		- (www.)cimbbank.com.my				(timeout)
```
Comment (J0WI) [inline src/chrome/content/rules/CIMB-Malaysia.xml]: WFM

File: src/chrome/content/rules/CIMB-Malaysia.xml [hunk 2]
```diff
@@ -22,21 +13,20 @@ Fetch error: http://www.cimb-principal.com.my/ => https://www.cimb-principal.com
...
+		- (www.)cimb-principal.com.my		(SSL error)
+		- (www.)cimb.com					(timeout)
```
Comment (J0WI) [inline src/chrome/content/rules/CIMB-Malaysia.xml]: WFM

**Comment by J0WI** [comment]
@youdly would you like to continue on this PR?

**Pull request closed without merging**